src/calculateRentalCost.js

```
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 3) {
+    return 100;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 7) {
+    return 230;
+  }
+
+  return 80;
```

This implementation doesn't follow the logic described in the task. It returns hardcoded values for specific inputs (`3`, `6`, `7`) but fails for others. For instance, `calculateRentalCost(1)` should return `40`, but your function currently returns `80`.

The logic should be:
1.  Calculate the base cost: `days * 40`.
2.  Apply a `$50` discount if `days >= 7`.
3.  Apply a `$20` discount if `days >= 3` (but less than 7).

This approach also violates checklist item #2, which advises against using magic numbers. You should define named constants for the daily rate, discount thresholds, and discount amounts.